infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/EnumTypeDefinition.java

```diff
@@ -134,9 +135,10 @@ public EnumTypeBuilder(final Class<T> itemType, final Function<T, String> serial
       this.serializer = serializer;
     }
 
-    public EnumTypeBuilder(final Class<T> itemType) {
```

Comment:
Maybe we could leave this one (default, do nothing) and add the new constructor with the flag